### DISCUSSION TITLE
Proposal for name change: QA ➡community

### DISCUSSION THREAD

**Discussion by pattacini** [robotology/community#448] [Feedback & Meta]
Given https://github.com/robotology/QA/discussions/447#discussioncomment-163751, I propose to change the name of the repo from `robotology/QA` to `robotology/community` as, through the mechanism of Discussions, we are now able to host lots more than just the Q&A system.

Also, consider that our community on Spectrum https://spectrum.chat/icub will soon need to be ported here because the team behind Spectrum has now joined GitHub Discussions.

Certainly, the change of the name will require some particular care at the beginning to avoid confusion and, most of all, to update links everywhere (redirection is guaranteed by GitHub anyway). I will deal with that and we can complete the process at our convenient own pace.

Other impacts stemmed from the use of the Discussions will be regarded for example with the way we currently generate announcements and highlights in https://robot-bazaar.iit.it/announcements @vtikha @Nicogene @AlexAntn 

cc @robotology/icub-developers @robotology/yarp-developers

**Comment by traversaro** [comment]
I fully agree! Considering all this restructuring, I think it would be nice if we could find a way for people to just subscribe (i.e. receive a notification on GitHub) to our announcement. At the moment it is only possible to subscribe (i.e. get a notification) whenever there is a new issue in robotology/QA, and that may be too much for people that are just interested in announcements.

**Reply by pattacini** [reply]
It would really great, but I'm afraid that such a subscription to specific categories is not available yet.

**Comment by drdanz** [comment]
I never really liked the QA name, but changing it now it will break lots of links in issues and documentation, and it's going to break some workflows (both manual and automated). Just searching "robotology/QA" in github, this is the result:

![image](https://user-images.githubusercontent.com/1100056/101602659-bec52680-39fe-11eb-811c-ea03574fa4ff.png)

Also, the name `community` does not really thrill me, perhaps `robotology-community`? Anyway I see only extra work coming from this change, with no real advantage, hence I'd start using the discussions and see how it goes. We can save this idea for later...

**Reply by pattacini** [reply]
`robotology/community` already subsumes the term `robotology`, plus links won't be really broken up as GitHub will make sure that redirection will be put in place. Obviously, refactoring/restyling will be required anyway for, let's say, beautifying the text. Thanks to the redirection we shouldn't be suffering from disruption.

At any rate, I agree we can keep this proposal in the backlog to cook for a while 👍🏻

**Comment by traversaro** [comment]
> I never really liked the QA name, but changing it now it will break lots of links in issues and documentation, and it's going to break some workflows (both manual and automated). Just searching "robotology/QA" in github, this is the result:

I think that GitHub renames work pretty well, you can still go to https://github.com/robotology/gazebo_yarp_plugins (with underscore, renamed to `gazebo-yarp-plugins` in early 2014) and it works fine, so I don't think that is a big problem in practice.

**Comment by pattacini** [comment]
My idea is to change `QA` into `community` during the upcoming holidays and proceed further with updating the text of the links when/where possible.

**Reply by pattacini** [reply]
cc @robotology/everyone

**Comment by pattacini** [comment]
Hi @robotology/everyone 👋🏻 

This is to let you know that [`robotology/QA`][1] has been just renamed into [`robotology/community`][2] to better reflect the intended use of such a repository.

Don't panic 😉 GitHub guarantees that old links – spread everywhere – are automatically redirected to the new place.

[1]: https://github.com/robotology/QA
[2]: https://github.com/robotology/community
